### PR TITLE
feat: tt token passport part 2

### PR DESCRIPTION
**Pull request by andrehadianto**
[Notion link](https://www.notion.so/sally-wallet/Part-2-of-2-Update-policy-and-Sally-screens-for-TT-Token-Passport-collection-collect-money-upon-1s-ded1eb59d6f74dc6be172fd69fef73ea) 

This PR adds... 
- `PAYMENT_RECEIPT` field to all `PASSPORT` input
- remove `tt-token-lost-waive` appeal block from `PASSPORT` entry

- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
- [ ] I've requested reviews from 1-2 reviewers
- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
- [ ] I've added/updated unit/integration tests
- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
- [ ] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
- [x] I've added documentation in README/[Notion](https://www.notion.so/82b92fb1007640328dab9582c0a3694e?v=3b6ce48202cf40ad8450553799b13146)

### CODE REVIEW

File: src/components/CustomerAppeal/CustomerAppealScreen.tsx [hunk 1]
```diff
@@ -108,25 +110,33 @@ export const CustomerAppealScreen: FunctionComponent<NavigationProps> = ({
...
+               * Some assumptions that are made:
+               * - Policies with dependencies on other policies should be defined after their dependents
+               * - Policies with dependencies show up depending on whether its dependent policy's
+               *   descriptionAlert is shown
+               */
+              const dependentReason = result.find(
+                (reason) =>
+                  reason.descriptionAlert && reason.category === policyThreshold
+              );
+              console.log(dependentReason);
```
Comment (dalsontws) [inline src/components/CustomerAppeal/CustomerAppealScreen.tsx:132]: rmbr to remove the `console.log` here!

File: src/components/CustomerAppeal/CustomerAppealScreen.tsx [hunk 2]
```diff
@@ -108,25 +110,33 @@ export const CustomerAppealScreen: FunctionComponent<NavigationProps> = ({
...
-            if (dependentReason && policy.alert) {
-              result.push({
-                category: policy.category,
-                description: policy.name,
-                descriptionAlert: policy.alert.label,
-              });
+            if (
+              !(
+                selectedIdType.validation === "PASSPORT" &&
+                policy.category === "tt-token-lost-waive"
```
Comment (dalsontws) [inline src/components/CustomerAppeal/CustomerAppealScreen.tsx:116]: do we still need to check if `typeof policyThreshold === "string"`?

File: src/components/CustomerQuota/utils.test.ts [hunk 3]
```diff
@@ -354,37 +354,45 @@ describe("pod related utils", () => {
...
 
   describe("isPodChargeable", () => {
-    it("should return true for tt-token-lost category", () => {
-      expect.assertions(1);
-      validCartItem["category"] = "tt-token-lost";
-      validCartItem["descriptionAlert"] = "*chargeable";
-      expect(
-        isPodChargeable(idType, validIdentifiers, validCartItem)
-      ).toStrictEqual(true);
+    beforeEach(() => {
```
Comment (dalsontws) [inline src/components/CustomerQuota/utils.test.ts:357]: will this still work if we use `beforeAll`?

File: src/components/CustomerQuota/utils.test.ts [hunk 4]
```diff
@@ -354,37 +354,45 @@ describe("pod related utils", () => {
...
+
+      it("should return false if there are no identifiers", () => {
+        expect.assertions(1);
+        expect(isPodChargeable(idType, [], validCartItem)).toStrictEqual(false);
+      });
     });
 
     describe("for passport customer", () => {
-      idType = "PASSPORT";
+      beforeEach(() => {
```
Comment (dalsontws) [inline src/components/CustomerQuota/utils.test.ts:393]: will this still work if we use `beforeAll`?

**Review by dalsontws — CHANGES_REQUESTED** [review]

File: src/components/CustomerAppeal/CustomerAppealScreen.tsx (hunk 2 above)
Comment (andrehadianto) [reply src/components/CustomerAppeal/CustomerAppealScreen.tsx:116]: yep still need! it checks if the threshold is dependant to other category or not!

File: src/components/CustomerQuota/utils.test.ts [hunk 5]
```diff
@@ -394,11 +402,11 @@ describe("pod related utils", () => {
         ).toStrictEqual(true);
       });
 
-      it("should return false for non-pod category", () => {
+      it("should return true for any category", () => {
         expect.assertions(1);
         expect(
           isPodChargeable(idType, validIdentifiers, validCartItem)
```
Comment (dalsontws) [inline src/components/CustomerQuota/utils.test.ts:408]: oh just noticed this, whats the difference between this test and the test for tt-token category? on line 397

Comment (andrehadianto) [reply src/components/CustomerQuota/utils.test.ts:408]: the new one will have "toilet-paper" as the category!

**Review by dalsontws — APPROVED** [review]
LGTM! 👍

**Pull request merged**